code/modules/clothing/shoes/magboots.dm

```diff
@@ -6,15 +6,18 @@
 	force = 3
 	overshoes = 1
 	var/magpulse = 0
+	var/mag_slow = 3
 	var/icon_base = "magboots"
 	action_button_name = "Toggle Magboots"
 	var/obj/item/clothing/shoes/shoes = null	//Undershoes
 	var/mob/living/carbon/human/wearer = null	//For shoe procs
+	armor = list(melee = 40, bullet = 30, laser = 30,energy = 25, bomb = 50, bio = 100, rad = 70)
```

Comment:
Magboots are big, heavy and slow as footwear goes, it seems reasonable to give them some protection. The slowdown is the obvious drawback